src/tracer/ISOMIP_tracer.F90

```diff
@@ -340,7 +343,7 @@ subroutine ISOMIP_tracer_column_physics(h_old, h_new,  ea,  eb, fluxes, dt, G, G
       if (fluxes%iceshelf_melt(i,j) > 0.0) then
 !        CS%tr(i,j,1,m) = (dt*fluxes%iceshelf_melt(i,j)/(365.0 * 86400.0)  &
 !                       + h_old(i,j,1)*CS%tr(i,j,1,m))/h_new(i,j,1)
-         CS%tr(i,j,1,m) = 1.0
+         CS%tr(i,j,1:3,m) = 1.0
```

Comment:
For now I just inject the tracer in the first tree layers. This will change
once we turn on the boundary layer scheme.

********************************************
Gustavo M. Marques

Postdoctoral Research Associate
http://gustavo-marques.com/
Princeton University/GFDL
201 Forrestal Road
Princeton, NJ 08540-6649

On Mon, Jun 27, 2016 at 3:31 PM, Alistair Adcroft <notifications@github.com>
wrote:

> In src/tracer/ISOMIP_tracer.F90
> <https://github.com/NOAA-GFDL/MOM6/pull/323#discussion_r68640670>:
>
> > @@ -340,7 +343,7 @@ subroutine ISOMIP_tracer_column_physics(h_old, h_new,  ea,  eb, fluxes, dt, G, G
> >        if (fluxes%iceshelf_melt(i,j) > 0.0) then
> >  !        CS%tr(i,j,1,m) = (dt*fluxes%iceshelf_melt(i,j)/(365.0 * 86400.0)  &
> >  !                       + h_old(i,j,1)*CS%tr(i,j,1,m))/h_new(i,j,1)
> > -         CS%tr(i,j,1,m) = 1.0
> > +         CS%tr(i,j,1:3,m) = 1.0
>
> What is the "1:3" for?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-GFDL/MOM6/pull/323/files/436369740e86db9c0ce1019d4c601b9eb6626dee#r68640670>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AK0FgZE8lVFXYvM2Pn1SBOZzDJW-3zVQks5qQCUggaJpZM4I7R_A>
> .
>